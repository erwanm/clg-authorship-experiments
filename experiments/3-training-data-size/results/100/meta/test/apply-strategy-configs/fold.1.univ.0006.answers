1894-fmc-wg 1906-fmc-alor	0.146516
1868-espw-tga 1879-haj-ttb	0.262652
1891-haj-tetb 1896-hj-toh	0
1887-fmc-s 1906-fmc-alor	0.249633
1888-fmc-si 1911-rwc-tcl	0.471679
1883-espw-btg 1887-espw-jtf	0.610384
1885-fmc-zamc 1898-fmc-ari-v2	0.045577
1888-fmc-si 1889-fmc-g	0.707113
1877-hj-ta 1883-hj-rh-v2	0.351323
1867-espw-gyatgc 1869-mt-ia	0.428634
1862-hbs-tpooi 1875-hbs-waon	0.591722
1882-haj-bn 1898-haj-abf	0.700015
1881-hj-tpoal 1917-hj-tit	0.673313
1880-hj-c 1898-hj-ttm	0.831048
1890-hj-ttm 1917-hj-tsotp	0.735183
1891-haj-tetb 1899-haj-jtpb	0.48009
1854-hbs-smofl-v2 1909-rwc-tdm	0.309658
1879-hj-h 1904-hj-tgb	0.625463
1903-us-tjoas 1917-us-kc	0.507883
1868-lma-lw 1881-mt-tpatp	0.382122
1888-fmc-si 1908-fmc-tp	0.34724
1873-wta-tyc 1876-wta-vao	0.287864
1876-haj-btlb 1882-haj-ffbts	0.361776
1902-fmc-c 1911-us-alp	0.084347
1866-haj-ccc-ia 1898-haj-tyb	0.180807
1886-hj-tb 1898-hj-itc	0.402559
1900-espw-wtg 1900-haj-adoh	0.235383
1893-fmc-md 1896-fmc-c	0
1883-espw-btg 1896-espw-tsasa	0.085289
1885-fmc-tcotk 1903-fmc-thor	0.142993
1898-hj-itc 1906-rwc-ttolp	0
1871-hbs-ofs 1875-hbs-waon	0.189503
1879-hj-h 1893-fmc-kl	0.46421
1885-hj-altof 1897-hj-wmk	0.363156
1886-hj-tb 1921-us-tbol	0.596334
1885-lma-ll 1897-ewaoc-tdof	0
1898-us-apom 1905-fmc-fm	0.203507
1888-hj-tr 1900-espw-wtg	0.301475
1870-haj-rar 1888-haj-bb	0.37608
1884-wta-Sac 1890-wta-otb	0.84404
1896-hj-toh 1903-fmc-thor	0.422233
1872-mt-ri 1892-fmc-pg	0.250792
1879-haj-tym 1882-haj-ffbts	0.305575
1903-ew-ivatg 1910-ab-cn	0.118963
1885-es-tpod 1915-rwc-b	0.221359
1874-wta-tcw 1905-es-tpoe	0.358529
1878-cdw-hikab 1910-rwc-ap	0.781227
1870-hbs-tbc 1871-hbs-pawt	0.66777
1897-rwc-l 1907-espw-tldup	0
1884-ab-sbts 1897-rwc-l	0.348502
1910-rwc-ap 1912-rwc-tsoa	0.281939
1854-hbs-smofl 1859-hbs-tmw	0.484571
1898-fmc-vc 1907-es-tlotg	0.518863
1871-espw-tsp 1896-hj-tsop	0.655623
1884-fmc-aap 1896-fmc-c	0.276243
1880-hj-c 1891-haj-tetb	0.379063
1908-fmc-tp 1917-us-kc	0.20827
1869-espw-mwag 1901-espw-a	0.814458
1874-cfw-tac 1891-cdw-oi	0.441694
1898-haj-abf 1899-haj-mmv	0.597666
1868-hbs-moot 1895-fmc-t	0
1866-wta-ob 1876-wta-vao	0.33647
1906-mt-cfma 1914-mh+-mhce-nf	0.612538
1864-lma-opdaot 1886-hj-pc	0.325505
1887-fmc-s 1917-us-kc	0.347895
1885-fmc-tcotk 1890-fmc-acmr	0.441248
1873-hbs-wish 1910-rwc-ap	0.493386
1870-haj-rar 1900-haj-adoh	0.651295
1893-fmc-kl 1922-rwc-tfj	0.247248
1854-hbs-smofl 1874-haj-rftr	0.25994
1868-haj-rd 1914-mh+-mhce-nf	0.294844
1866-haj-ccc-ia 1882-haj-bn	0.535527
1895-fmc-t 1896-espw-cfal	0.589159
1898-haj-tyb 1899-haj-aoatb	0.426506
1852-hbs-utc 1902-fmc-c	0.417705
1893-fmc-kl 1898-fmc-vc	0.485566
1909-rwc-tdm 1919-ew-m	0.386543
1884-fmc-aap 1892-fmc-pg	0.588126
1870-haj-rar 1895-ab-mgtonel	0.479825
1869-wta-dtr 1870-wta-faf	0.64936
1898-fmc-ari 1906-fmc-a	0.482708
1858-wta-pap 1866-wta-ob	0.950132
1881-hj-tpoal 1897-hj-wmk	0.813558
1862-hbs-tpooi 1895-wta-alae	0.270016
1883-fmc-dc 1907-fmc-tdr	0.54068
1862-hbs-tpooi 1871-espw-tsp	0.449716
1878-hj-te 1903-hj-ta	0.526965
1889-wta-wtel 1905-es-tpoe	0.047004
1864-wta-tsb 1890-fmc-acmr	0.768687
1872-wta-lb 1905-fmc-fm	0.396239
1884-mt-taohf 1893-fmc-kl-v2	0.641679
1889-espw-tsfi 1895-fmc-ajs	0.442646
1885-fmc-zamc 1907-fmc-tdr	0.808016
1862-hbs-aos 1883-wta-sh	0.416917
1891-haj-tetb 1898-fmc-ari	0
1897-haj-fof 1900-haj-adoh	0.568754
1914-us-sm 1917-us-kc	0.682554
1884-wta-Sac 1899-haj-aoatb	0.755343
1852-hbs-utc 1873-hbs-pl	0.59233
1864-wta-tsb 1874-mt-lotm	0.518277
