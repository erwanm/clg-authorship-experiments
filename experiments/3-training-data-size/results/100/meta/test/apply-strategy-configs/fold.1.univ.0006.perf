0.397800	0.663000	0.600000
