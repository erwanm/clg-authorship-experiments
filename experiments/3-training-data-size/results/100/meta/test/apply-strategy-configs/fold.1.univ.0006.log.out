apply-config.sh: applying config '/tmp/expe3/100/process/selected-meta-configs/fold.1.0098.model/strategy-configs/fold.1.univ.0006' to cases '/home/moreaue/dump/expe3//results/100/meta/test/apply-strategy-configs/only-cases.tmp.R4tRDbFA'; results written to '/home/moreaue/dump/expe3//results/100/meta/test/apply-strategy-configs/fold.1.univ.0006'
apply-config.sh: done.
